buildenv

```diff
@@ -1,20 +1,13 @@
-export ZOPEN_GIT_URL="https://github.com/direnv/direnv.bit"
-export ZOPEN_GIT_DEPS="comp_goport make"
+export ZOPEN_GIT_URL="https://github.com/direnv/direnv.git"
+export ZOPEN_GIT_DEPS="comp_go make coreutils"
 export ZOPEN_TYPE="GIT"
 export ZOPEN_RUNTIME_DEPS="bash"
 
 export ZOPEN_CONFIGURE="skip"
 export ZOPEN_MAKE="make"
 export ZOPEN_MAKE_MINIMAL="yes"
-export ZOPEN_CHECK="skip"
-export ZOPEN_INSTALL="zopen_install"
-
-zopen_init()
-{
-  # Go installs binaries, so create the bin dir as well
-  export GOBIN=$ZOPEN_INSTALL_DIR/bin
-  mkdir -p \$ZOPEN_INSTALL_DIR/bin
-}
+export ZOPEN_CHECK_OPTS="-i test"
+export ZOPEN_INSTALL_OPTS="install PREFIX=\$ZOPEN_INSTALL_DIR"
```

Comment:
The `make install` from the project is doing a few more things which we can't support at the moment; hence the `go install`.

Ref: https://github.com/direnv/direnv/blob/629ab91ed3c55f2380429e2a5e734d7cc502fb29/GNUmakefile#L154

- It adds a hook for the fish shell, which we don't have.
- For making `man` pages, it depends on `groff` and github.com/cpuguy83/go-md2man. We don't have `go-md2man` but can maybe port if it's generally useful.

Reply:
Although this `make install` will put the final binary to `...\bin`, we can't use `make install`.
However, a bare `make` works.